### DISCUSSION TITLE
Where does SteamCMD download to?

### DISCUSSION THREAD

**Discussion by JakeShirley** [lloesche/valheim-server-docker#127] [Q&A]
Hi!

Recently I have been hitting issues with my docker image running out of space.  This container is one of the recent installs so I thought I would inquire if it might be causing it.

One thought I had was the SteamCMD downloads.   Are these downloaded within  the container? If so would it be possible to expose a path for external binding so that I could dump them to disk elsewhere?

**Comment by lloesche** [comment]
The Dockerfile and the README list two volumes. One at `/config` for the config and one at `/opt/valheim` for the data which amounts to around 3 GB. Supervisord will write some logs to /var/log/supervisord but those are on the order of single digit kb per day.
